content/posts/newsletter/2021-05-05.md

```diff
@@ -0,0 +1,207 @@
+
+## XSF Announcement
+
+The XMPP Standards Foundation is [now on YouTube](https://www.youtube.com/channel/UCf3Kq2ElJDFQhYDdjn18RuA)! You can find recordings of conferences, the XMPP Office Hours, and other fun stuff over on the channel.
+
+Have a talk, demo, or roundtable discussion about XMPP or related technologies? XMPP Office Hours are weekly meetings of XMPP enthusiasts for discussion and presentations. For more information, or to sign up, have a look at [the wiki page](https://wiki.xmpp.org/web/XMPP_Office_Hours).
+
+## Events
+
+[XMPP Office Hours each week](https://wiki.xmpp.org/web/XMPP_Office_Hours) - Also, checkout our new [YouTube channel](https://www.youtube.com/channel/UCf3Kq2ElJDFQhYDdjn18RuA)!
+
+[Berlin XMPP Meetup (remote)](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup): Monthly Meeting of XMPP Enthusiasts in Berlin - always 2nd Wednesday of every month.
+
+## Articles 
+
+From May 2021, there have been major content updates for [freie-messenger.de](https://www.freie-messenger.de) as well as additions. From now on, the content will be internationalised. As a first step, there is the new [Quick Overview of Messengers](https://www.freie-messenger.de/systemvergleich/#schnellübersicht) in both German and English. Call for supporters!
+
+[Isode presents their military capabilities at DSEI 2021](https://www.isode.com/company/wordpress/dsei-2021-demo/), offering Military Instant Messaging and more.
+
+[Gajim Development News](https://gajim.org/post/2021-04-30-development-news-april/): April brought an all new message styling parser, making Gajim fully compliant with XEP-0393. This post will also give you a sneak peek on some features we’ve been developing for the past months: the new Chat view and Contact Info window.
+
+![Gajim chat view](/images/newsletter/2021-04/gajim_chat-view-april.png "Gajim chat view")
+
+Libervia (formerly Salut à Toi) has been selected for a grant by NLNet/NGI0 Discovery Fund (with financial support from European Commission's Next Generation Internet programme) for an [ActivityPub <=> XMPP 
+gateway doubled with Pubsub end-to-end encryption project](https://www.goffi.org/b/activitypub-gateway-and-pubsub-e2ee-QGqK). The XMPP <=> ActivityPub gateway will join two major open and decentralised protocols. In practice it will be a XMPP server component (usable with any server), and implement the ActivityPub server to server protocol (or "Federation Protocol"). On XMPP side, it will be mostly a Pubsub service (with some extra, like private messages converted to XMPP message stanza).
+
+[Vaxbot continues to grow](https://monal.im/blog/vaxbot-continues-to-grow/): VaxBot is a free service that helps you find vaccine appointments in your state or country if available. During the last weeks, [Vaxbot](vaccine.monal.im) continued to grow. The service just hit 8 million messages sent and it sends around 800,000 messages a day. XMPP and 
```

Comment:
maybe add an "https://" here - is a little more beautiful
shouldn't make a difference in the generated html, but in github preview it messed the link up due to the missing protocol